pandas/io/formats/format.py

```diff
@@ -1810,11 +1823,23 @@ class Datetime64TZFormatter(Datetime64Formatter):
     def _format_strings(self) -> list[str]:
         """we by definition have a TZ"""
         values = self.values.astype(object)
+        values_shape = values.shape
+        values = values.ravel() if values.ndim > 1 else values
         ido = is_dates_only(values)
         formatter = self.formatter or get_format_datetime64(
             ido, date_format=self.date_format
         )
-        fmt_values = [formatter(x) for x in values]
+
+        if len(values_shape) > 1:
```

Comment:
might be worth making a helper function ofr L1833-1842 so can call here & in the super method